linux_os/guide/system/auditing/auditd_configure_rules/audit_dac_actions/audit_rules_dac_modification_lsetxattr/rule.yml

```diff
@@ -70,7 +70,8 @@ references:
     stigid@ol8: OL08-00-030220
     stigid@rhel7: RHEL-07-030440
     stigid@rhel8: RHEL-08-030200
-    stigid@sle15: SLES-15-030240
+    stigid@sle15: SLES-12-020370
+    stigid@sle15: SLES-15-030190
```

Comment:
I'm sorry for the confusion. It is allowed to have multiple references, but it isn't allowed to have multiple same keys. The problem seems to be here. There are 2 YAML keys `stigid@sle15`.

Reply:
Naturally @jan-cerny thanks for bringing this up, I guess I missed to rung the `fix_rules-duplicate_subkeys` test on my setup before pushing :bow: 
Should be ok now in [9aa460c](https://github.com/ComplianceAsCode/content/pull/8409/commits/9aa460c496b2b54d0bbfd17762774451e93e7b47)